### PR TITLE
Add schedule for minimal+minimal_role for PowerVM

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
The PR adds new yaml schedule for minimal+minimal_role on PowerVM.

- Related ticket: https://progress.opensuse.org/issues/68736
- Verification run: https://openqa.suse.de/tests/4441263
- **Job Group Settings:** https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/238

### CODE REVIEW

File: schedule/yast/minimal+role_minimal/minimal+role_minimal_pvm.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/installation_overview
+  - installation/edit_optional_kernel_cmd_parameters
```
Comment (rwx788) [inline schedule/yast/minimal+role_minimal/minimal+role_minimal_pvm.yaml:23]: As you didn't define `OPT_KERNEL_PARAMS`, so it doesn't make sense to schedule this module. Without this module, also feel free to unify both schedules.

**Comment by jknphy** [comment]
is any validation module that makes sense to add this scenario? From AC1 in https://progress.opensuse.org/issues/68736 I'm not sure if it is referring to existing one or to create new one if need it. Perhaps there is some module that check the patterns with test data. It is true that for the description of the test suite seems not to be necessary, wdty?

**Comment by rwx788** [comment]
> is any validation module that makes sense to add this scenario? From AC1 in https://progress.opensuse.org/issues/68736 I'm not sure if it is referring to existing one or to create new one if need it. Perhaps there is some module that check the patterns with test data. It is true that for the description of the test suite seems not to be necessary, wdty?

I copy paste content of the tickets, so if validation is not there, we should not add in a scope of this ticket.

File: schedule/yast/minimal+role_minimal/minimal+role_minimal_pvm.yaml (hunk 1 above)
Comment (OleksandrOrlov) [reply schedule/yast/minimal+role_minimal/minimal+role_minimal_pvm.yaml]: Removed the extra module and unified schedule.

**Pull request merged**